Readme.md

```diff
@@ -1,11 +1,6 @@
 # Lombiq Privacy for Orchard Core
 
-
-
-[![Lombiq.Privacy NuGet](https://img.shields.io/nuget/v/Lombiq.Privacy?label=Lombiq.Privacy)](https://www.nuget.org/packages/Lombiq.Privacy/)
-[![Lombiq.Privacy.Samples NuGet](https://img.shields.io/nuget/v/Lombiq.Privacy.Samples?label=Lombiq.Privacy.Samples)](https://www.nuget.org/packages/Lombiq.Privacy.Samples/)
-[![Lombiq.Privacy.Tests.UI NuGet](https://img.shields.io/nuget/v/Lombiq.Privacy.Tests.UI?label=Lombiq.Privacy.Tests.UI)](https://www.nuget.org/packages/Lombiq.Privacy.Tests.UI/)
-
+[![Lombiq.Privacy NuGet](https://img.shields.io/nuget/v/Lombiq.Privacy?label=Lombiq.Privacy)](https://www.nuget.org/packages/Lombiq.Privacy/) [![Lombiq.Privacy.Samples NuGet](https://img.shields.io/nuget/v/Lombiq.Privacy.Samples?label=Lombiq.Privacy.Samples)](https://www.nuget.org/packages/Lombiq.Privacy.Samples/) [![Lombiq.Privacy.Tests.UI NuGet](https://img.shields.io/nuget/v/Lombiq.Privacy.Tests.UI?label=Lombiq.Privacy.Tests.UI)](https://www.nuget.org/packages/Lombiq.Privacy.Tests.UI/)
```

Comment:
The previous formatting is so much more readable - is there a way to keep like that with some ignore rule?

Reply:
The logic is that if something is to be rendered in one paragraph then it should be one line in the source too. I completely agree with it, and if we start making exceptions it will birth infinite arguments in the future.

Reply:
OK! So be it.